tools/lsp/preview.rs

```diff
@@ -658,23 +655,21 @@ fn resize_selected_element(x: f32, y: f32, width: f32, height: f32) {
     send_workspace_edit(label, edit);
 }
 
-fn resize_selected_element_impl(
-    rect: LogicalRect,
-) -> common::Result<Option<(lsp_types::WorkspaceEdit, String)>> {
+fn resize_selected_element_impl(rect: LogicalRect) -> Option<(lsp_types::WorkspaceEdit, String)> {
     let Some(selected) = selected_element() else {
-        return Ok(None);
+        return None;
```

Comment:
looks like all of these return in that function could be changed to `?`

Reply:
D'oh! Of course :-)